roles/map-role/templates/pf9-ostackhost-neutron.j2

```diff
@@ -1,5 +1,8 @@
 {
     "cluster_ip": "{{ha_cluster_ip}}",
     "instances_path": "{{nova_instances_path}}",
-    "novncproxy_base_url": "{{neutron_novncproxy_base_url}}"
+    "novncproxy_base_url": "{{neutron_novncproxy_base_url}}",
```

Comment:
Have not fully vetted but won't this comma at the end of the line cause invalid json to be submitted to resmgr if sriov is not on, which would cause non-sriov deployments to fail.

Reply:
I don't recall the behavior, but i moved the SR-IOV bit to avoid an issue.